content/cookbook/wallets/restore-keypair.md

```diff
@@ -22,15 +45,43 @@ const keypair = Keypair.fromSecretKey(
 );
 ```
 
+</Tab>
+
+</Tabs>
+
 ## From base58 String
 
-```typescript filename="restore-keypair-from-base58.ts
-import { Keypair } from "@solana/web3.js";
-import * as bs58 from "bs58";
+<Tabs groupId="language" items={['web3.js v2', 'web3.js v1']}>
 
-const keypair = Keypair.fromSecretKey(
+<Tab value="web3.js v2">
+
+```typescript
+import { createKeyPairFromBytes } from "@solana/web3.js";
+import bs58 from "bs58";
+
+const keypair = await createKeyPairFromBytes(
   bs58.decode(
     "5MaiiCavjCmn9Hs1o3eznqDEhRwxo7pXiAYez7keQUviUkauRiTMD8DrESdrNjN8zd9mTmVhRvBJeg5vhyvgrAhG",
   ),
 );
+
+console.log(keypair);
 ```
+
+</Tab>
+
+<Tab value="web3.js v1">
+
+```typescript
+import { createKeyPairFromBytes, getBase58Encoder } from "@solana/web3.js";
+
+const keypair = await createKeyPairFromBytes(
+    getBase58Encoder().decode("5MaiiCavjCmn9Hs1o3eznqDEhRwxo7pXiAYez7keQUviUkauRiTMD8DrESdrNjN8zd9mTmVhRvBJeg5vhyvgrAhG")
+);
+
+console.log(keypair);
+```
```

Comment:
It looks like you might have accidently deleted the v1 snippet here when updating to use `getBase58Encoder()`